common/src/main/java/org/jboss/hal/testsuite/page/runtime/WebMetricsPage.java

```diff
@@ -7,9 +7,9 @@
 /**
  * Created by mkrajcov on 4/10/15.
  */
-@Location("#web-metrics")
+@Location("#http-metrics")
```

Comment:
IMO this `@Location` annotation can be removed. Anyway neither "#web-metrics" nor "#http-metrics" should be used.